docs/build/html/gettingstarted.html

```diff
@@ -0,0 +1,285 @@
+              <p class="caption"><span class="caption-text">Contents</span></p>
+<ul class="current">
+<li class="toctree-l1 current"><a class="current reference internal" href="#">Getting Started</a><ul>
+<li class="toctree-l2"><a class="reference internal" href="#requirements">Requirements</a></li>
+<li class="toctree-l2"><a class="reference internal" href="#install-hogarama">Install Hogarama</a></li>
+<li class="toctree-l2"><a class="reference internal" href="#upgrade-hogarama">Upgrade Hogarama</a></li>
+<li class="toctree-l2"><a class="reference internal" href="#uninstall-hogarama">Uninstall Hogarama</a></li>
+<li class="toctree-l2"><a class="reference internal" href="#additional-commmands">Additional Commmands</a></li>
+</ul>
+</li>
+<li class="toctree-l1"><a class="reference internal" href="habarama.html">Habarama</a></li>
+<li class="toctree-l1"><a class="reference internal" href="hogajama.html">Hogajama</a></li>
+<li class="toctree-l1"><a class="reference internal" href="documentation.html">Documentation</a></li>
+</ul>
+
+            
+          
+        </div>
+      </div>
+    </nav>
+
+    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">
+
+      
+      <nav class="wy-nav-top" aria-label="top navigation">
+        
+          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
+          <a href="index.html">Hogarama</a>
+        
+      </nav>
+
+
+      <div class="wy-nav-content">
+        
+        <div class="rst-content">
+        
+          
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+<div role="navigation" aria-label="breadcrumbs navigation">
+
+  <ul class="wy-breadcrumbs">
+    
+      <li><a href="index.html">Docs</a> &raquo;</li>
+        
+      <li>Getting Started</li>
+    
+    
+      <li class="wy-breadcrumbs-aside">
+        
+            
+            
+              <a href="https://github.com/gepardec/Hogarama/blob/master/docs/source/gettingstarted.rst" class="fa fa-github"> Edit on GitHub</a>
+            
+          
+        
+      </li>
+    
+  </ul>
+
+  
+  <hr/>
+</div>
+          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
+           <div itemprop="articleBody">
+            
+  <div class="section" id="getting-started">
+<h1>Getting Started<a class="headerlink" href="#getting-started" title="Permalink to this headline">¶</a></h1>
+<div class="section" id="requirements">
+<h2>Requirements<a class="headerlink" href="#requirements" title="Permalink to this headline">¶</a></h2>
+<ul class="simple">
+<li><p>OpenShift 4.0 or above</p></li>
+<li><p>Docker</p></li>
+<li><p>Our infrastructure secrets</p></li>
+</ul>
+<p>In order to run Hogarama you need access to an OpenShift 4.0 cluster or above and to be able to run docker on your workstation. In addition to that you need our infrastructure secrets. The deployment script checks if it can locate the secrets file on your workstation and if it can’t find it, it will tell you where to get it from.</p>
```

Comment:
Looks like generated code and shouldn't be  checked in.

Reply:
yes that is generated code. Due to the way GitHub pages work I am not aware of any other way to serve documentation as github pages in the same manner as readthedocs.io does it without directly utilizing readthedocs as a hosting provider.